index.js

```diff
@@ -131,7 +139,9 @@ const destroyCircular = ({
 		for (const {property, enumerable} of errorProperties) {
 			if (from[property] !== undefined && from[property] !== null) {
 				Object.defineProperty(to, property, {
-					value: isErrorLike(from[property]) ? continueDestroyCircular(from[property]) : from[property],
+					value: isErrorLike(from[property]) || Array.isArray(from[property])
```

Comment:
```suggestion
					value: property === 'cause' || property === 'errors'
```

Reply:
This is a potentially breaking change though. And not directly related to this pull request. I would prefer it to be a separate pull request and also, `cause` is (unfortunately) not required to be an error. There should be a test for that.

Reply:
Yeah it breaks when `cause` is not an `object`.

Merge as is